Kudu.Services.Web/Content/Scripts/ProcessExplorer.js

```diff
@@ -868,6 +894,50 @@ function searchForHandle() {
     }
 }
 
+function handleProfilingEvents(e, processId) {
+    if (!e || !e.target || typeof e.target.textContent === "undefined") {
+        return;
+    }
+
+    if (e.target.textContent.indexOf("Stop") === 0) {
+        stopProfiling(e, processId);
+    }
+    else if(e.target.textContent.indexOf("Starting") !== 0) {
+        e.target.textContent = "Starting Profiler...";
+        startProfiling(e, processId);
+    }
+}
+
+function postProfileRequest(action, processId) {
+    var uri = "/api/processes/" + processId + "/profile/" + action;
+    var request = {
+            method: "POST",
+            contentType: "application/json",
+        };
+    return $.ajax(uri, request);
+}
+
+function startProfiling(e, processId) {
+    var request = postProfileRequest("start", processId);
+    request.done(function (resp) {
+        e.target.title = "";
+        e.target.textContent = "Stop Profiling..";
+    });
+    request.fail(function (resp) {
+        var obj = JSON.parse(resp.responseText);
+        alert(obj["Message"]);
+        e.target.textContent = "Start Profiling..";
```

Comment:
Why are there ellipsis on this text? I guess it'd help to see the UI so I'm clear what this is doing. From looking here, it looks like this should just be "Start Profiling" without ellipsis?